packages/api/tests/mock.factories.js

```diff
@@ -1150,6 +1150,10 @@ async function soil_amendment_purposeFactory() {
   return knex('soil_amendment_purpose').insert({ key: faker.lorem.word() }).returning('*');
 }
 
+async function soil_amendment_fertiliser_typeFactory() {
+  return knex('soil_amendment_purpose').insert({ key: faker.lorem.word() }).returning('*');
```

Comment:
```suggestion
  return knex('soil_amendment_fertiliser_type').insert({ key: faker.lorem.word() }).returning('*');
```

Reply:
and I think this table `soil_amendment_fertiliser_type` needs to be added to `tableCleanup`

Reply:
Thanks!